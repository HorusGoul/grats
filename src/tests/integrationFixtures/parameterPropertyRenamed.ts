/** @gqlType */
export class Query {
  constructor(
    /** @gqlField hello */
    public NOT_THIS: string = "world",
  ) {}
}

export const query = `
  query {
    hello
  }
`;
