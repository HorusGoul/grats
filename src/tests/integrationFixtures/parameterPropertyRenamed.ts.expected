-----------------
INPUT
----------------- 
/** @gqlType */
type Query = unknown;

/** @gqlField hello */
export function NOT_THIS(_: Query): string {
  return "world";
}

export const query = `
  query {
    hello
  }
`;

-----------------
OUTPUT
-----------------
{
  "data": {
    "hello": "world"
  }
}